Can't use variables in static tag (dynamic assets)

Hi, first of all, I would like to thank you. Nice repo!
Is this a bug or am I doing something wrong?

## Can't use variables in static tag (dynamic assets)
```javascript
{% for image in images %}
  <img src="{% static image %}">
{% endfor %}
```
```
Error: Child compilation failed:
  Module build failed (from ../node_modules/simple-nunjucks-loader/lib/cjs/loader.js):
  Error: Asset "image" not found
```
**or**
```javascript
{% for image in images %}
  <img src="{% static './' + image %}">
{% endfor %}
```
```
Error: Expected html to be a string but got null
  
  - index.js:417 HtmlWebpackPlugin.postProcessHtml
    [website]/[html-webpack-plugin]/index.js:417:29
  
  - index.js:247 
    [website]/[html-webpack-plugin]/index.js:247:25
  
  - task_queues.js:97 processTicksAndRejections
    internal/process/task_queues.js:97:5
```

## Expected behavior
We have a list of images:
```javascript
new HTMLWebpackPlugin({
    template: 'template.njk',
    templateParameters: {
        images: ['first.jpg', 'second.jpg', 'third.jpg']
    }
})
```
And I want to use them in the template:
```javascript
{% for image in images %}
  <img src="{% static image %}">
{% endfor %}
```


## Config
- Windows 10
- NodeJS - 12.16.3
- file-loader - 6.0.0
- nunjucks - 3.2.2
- simple-nunjucks-loader - 2.0.2

### Plugin
```javascript
new HTMLWebpackPlugin({
    template: 'template.njk',
    templateParameters: {
        images
    }
})
```

### Loader
```javascript
{
  test: /\.(njk|nunjucks|html)$/i,
  include: src,
  use: [
    'cache-loader',
    {
      loader: 'thread-loader',
      options: { workerParallelJobs: 100, poolRespawn: false, name: 'nunjucks' }
    },
    {
      loader: 'simple-nunjucks-loader',
      options: {
        searchPaths: ['src'],
        assetsPaths: ['src/images']
      }
    }
  ]
},
{
  test: /\.(png|jpe?g|gif)$/i,
  use: ['file-loader']
}
```

That was my second thought:
```javascript
{% for image in images %}
  <img src="{% static './' + image %}">
{% endfor %}
```
```
Error: Expected html to be a string but got null
```

and tried to put images in a subfolder:
```javascript
{% for image in ['first.jpg', 'second.jpg'] %}
  <img src="{% static 'images/' + image %}">
{% endfor %}
```
```
Error: Cannot find module './imagesfirst.jpg'
```
or
```javascript
{% for image in ['/first.jpg', '/second.jpg'] %}
  <img src="{% static 'images/' + image %}">
{% endfor %}
```
```
Error: Cannot find module './images/first.jpg'
```

I'm afraid loader lack of support for "dynamic-only" assets. I need to change assets output format and the way i examine that it is actually dynamic and not a static path. I guess i will make path on weekend.

I will fix a trailing slash problem for now (`Cannot find module './imagesfirst.jpg'`), i miss that case when write tests for assets.

--- a/src/output/get-assets.js
+++ b/src/output/get-assets.js
@@ -21,10 +21,16 @@ export function getAssets(assets) {
                 importPath = assetImport.split(' + ').map(
                     function(part) {
                         if (part.startsWith('"')) {
-                            return stringifyRequest(
+                            let nextPart = stringifyRequest(
                                 loaderContext,
                                 part.replace(/^"|"$/g, '')
                             );
+
+                            if (!nextPart.endsWith('/"') && part.endsWith('/"')) {
+                                nextPart = nextPart.replace(/"$/, '/"');
+                            }
+
+                            return nextPart;
                         }
 
                         return part;

--- a/src/utils/resolve-search-paths.js
+++ b/src/utils/resolve-search-paths.js
@@ -3,7 +3,11 @@ import {unquote} from './unquote';
 
 function getFilePath(searchPath, possiblePath) {
     const [firstPart, ...restParts] = possiblePath.split(' + ');
-    const filePath = path.resolve(searchPath, unquote(firstPart));
+    let filePath = path.resolve(searchPath, unquote(firstPart));
+
+    if (firstPart.endsWith('/"') && !filePath.endsWith('/')) {
+        filePath = `${filePath}/`;
+    }
 
     return restParts.length > 0 ?
         `${[`"${filePath}"`, ...restParts].join(' + ')}` : filePath;

--- a/test/assets.test.js
+++ b/test/assets.test.js
@@ -1,3 +1,5 @@
+/* globals __USE_ES__ */
+
 import compiler from './compiler';
 
 const loaderOptions = {
@@ -22,6 +24,12 @@ describe('Assets', function() {
 
     test('should load dynamic assets', async function() {
         const output = await compiler('fixtures/assets/dynamic.njk', loaderOptions);
+
+        await expect(output()).resolves.toMatchSnapshot();
+    });
+
+    test('should load dynamic assets with trailing slashes', async function() {
+        const output = await compiler('fixtures/assets/dynamic-trailing-slash.njk', loaderOptions);
 
         await expect(output()).resolves.toMatchSnapshot();
     });

--- a/test/fixtures/assets/dynamic-trailing-slash.njk
+++ b/test/fixtures/assets/dynamic-trailing-slash.njk
@@ -1,0 +1,3 @@
+{% for fileName in ['dynamic-example-1.md', 'dynamic-example-2.md'] %}
+<p>{% static 'foo/bar/' + fileName %}</p>
+{% endfor %}

